app/views/directives/bind-service/results.html

```diff
@@ -0,0 +1,39 @@
+<div>
+  <div ng-if="!ctrl.error">
+    <div ng-if="(ctrl.binding | statusCondition : 'Ready').status !== 'True'">
+      <h3 class="mar-top-none center">
+        <span class="fa fa-spinner fa-pulse fa-3x fa-fw" aria-hidden="true"></span>
+        <span class="sr-only">Pending</span>
+        <div class="mar-top-lg">The binding was created but is not ready yet.</div>
+      </h3>
+    </div>
+    <div ng-if="(ctrl.binding | statusCondition : 'Ready').status === 'True'">
+      <h3 class="mar-top-none">
+        <strong>{{ctrl.serviceToBind}}</strong> has been bound to <strong>{{ctrl.target.metadata.name}}</strong> successfully
+      </h3>
+      <div class="sub-title">
+        <h3><span class="pficon pficon-ok mar-right-sm"></span>Configuration Created</h3>
+        The binding operation created the secret
+        <a ng-href="{{ctrl.generatedSecretName | navigateResourceURL : 'Secret' : ctrl.target.metadata.namespace}}">{{ctrl.generatedSecretName}}</a>
```

Comment:
Trying to decide if it's a worth a `canI` check here. Presumably if you can bind, you can view secrets.